apps/explorer/components/Block/index.tsx

```diff
@@ -5,30 +5,35 @@ import { useMemo } from 'react'
 import { routes } from '../../config/routes'
 import { EntityHeading } from '../EntityHeading'
 import { ContentLayout } from '../ContentLayout'
-import { SiaCentralBlock } from '@siafoundation/sia-central-types'
+import { ExplorerBlock } from '@siafoundation/explored-types'
 
 type Props = {
-  block: SiaCentralBlock
+  block: ExplorerBlock
+  blockID: string
 }
 
-export function Block({ block }: Props) {
-  const values = useMemo(() => {
-    const list: DatumProps[] = [
+export function Block({ block, blockID }: Props) {
+  const blockDatum: DatumProps[] = useMemo(() => {
+    // Grab the miner payout address
+    const minerPayoutAddress = block.minerPayouts.find(
+      (payout) => payout.source === 'miner_payout'
+    )?.siacoinOutput.address
+    // Trim "bid:" from the incoming blockID
+    const trimmedBlockID = blockID.slice(4)
```

Comment:
There is a method called `stripPrefix` you can use for this.

Reply:
We actually need to expand stripPrefix to support this case. Will do that.

Reply:
Done.